### PR TITLE
Merge Develop to Master for 1.0

### PR DESCRIPTION
**Pull request by iKrazy**
Readme got updated, script got updated.

### CODE REVIEW

**Comment by saitei** [comment]
@iKrazy you get a chance to test it all? I was just waiting for some feedback from the subreddit first, but if you're comfortable I am.  We can probably start to advertise it a bit wider if it's stable.  Should draw in some fresh coding blood.

**Comment by solifugo** [comment]
I just found an issue in RC1:

-- Vertcoin wallet points to Rates sheet instead of wallet:

  var vtcWallet = getVtcBalance("Vo8EXgAtxCVUtMaTQECuzLD2tZU1HqLbhT");
  ssRates.getRange('B5').setValue(vtcWallet);

--- Also, would be nice to populate Wallet sheet with some titles:

  var ethApiKey = "Your Etherscan API Key";
  var ethWallet = getEthBalance(ethApiKey,"0x7E9DDB5343a583705Ed9ADE065C0595EFB55D681");
  ssWallets.getRange('B4').setValue(ethWallet);
  ssWallets.getRange('A4').setValue("Your ETH Wallet:")

File: scripteditor.js [hunk 1]
```diff
@@ -1,105 +1,294 @@
...
-  //Change the value in getRange() to match the cell in spreadsheet
-  //Change the value in setValue() to match the variable above
+  //
+  // ===== BCH Wallet Balances =======
+  // Add more as needed with different variable names
+  // Change the value in getRange() to match the cell in spreadsheet
+  // Change the value in setValue() to match the wallet variable used (e.g. bchWallet)
+  // =================================
 
-  ssRates.getRange('E3').setValue(VtcWallet);
```
Comment (solifugo) [inline scripteditor.js]: VTC wallet points to Rates sheet instead of Wallet

Comment (saitei) [reply scripteditor.js]: 👍

**Comment by saitei** [comment]
Oddly enough, I did put code in to make column titles for the wallets but took it out.  I figured that since it wasn't auto-generating like rates that it would just cause more issues.  It would have to start commented out, then they'd have to uncomment that for three cells of text, etc.  I also have concerns about people not checking all of their cell references so they don't overlap. 

I also toyed with the idea of turning the camelCase wallet names into readable ones like in your example.  There are definitely some wallet related improvements that could be made.  Maybe create an issue to have it autogenerate that sheet as well.  Big pain right now for people to go through and manually change that stuff.  Short term measure could even be to make a generic function (e.g. `getBalance("cell", "currency", "address")`), and take care of the rest behind the scenes.

My biggest concern is still the API calls though.  The BTC ones were flaky because the server rpc queue was out of sync, the ETH one was failing due to checking too many in a row, etc.  It may be easier / more reliable to just scrape the values from a page instead of worrying about API issues.

**Pull request merged**